ngx_http_apicast_module.c

```diff
@@ -57,14 +58,28 @@ ngx_module_t  ngx_http_apicast_module = {
 };
 
 static ngx_http_apiast_ctx_t * ngx_http_apicast_set_ctx(ngx_http_request_t *r) {
+
```

Comment:
While you are at it, let's fix the typo `s/ngx_http_apiast_ctx_t/ngx_http_apicast_ctx_t`.